README.md

```diff
@@ -1,67 +1,58 @@
-# AirbrakesV2
-
+# Air BrakesV2 🚀
 
 ## Overview
-This project controls the extension of airbrakes using a servo motor. It includes modules for logging data and managing the servo motor.
-
-### Installation
-
-#### Install uv:
-
-[uv](https://docs.astral.sh/uv/getting-started/installation/) is a python and project manager, which makes handling different python versions and virtual enviornments easier.
```

Comment:
no `uv` :(

Reply:
I thought about it more -- while it's a nice thing, I don't think it's required/related specifically to our project. If someone wants to use it, that's great, but I don't think we should make them do it -- just trying to simplify installation